ledger/ledger-utxo/src/main/java/net/corda/v5/ledger/utxo/token/selection/TokenSelection.java

```diff
@@ -96,4 +96,14 @@ public interface TokenSelection {
     @Nullable
     @Suspendable
     TokenClaim tryClaim(@NotNull TokenClaimCriteria criteria);
+
```

Comment:
We need to review all of the comments on this class, they are written as if it only does selection, but now it does balance as well.
- We should rewrite the overall class level description
- Move the selection examples to the selection method
- Add a balance query example

Reply:
Done.